### PR TITLE
Add CentOS Stream 9 workers and builders

### PR DESCRIPTION
**Pull request by stratakis**

### CODE REVIEW

File: master/custom/workers.py [hunk 1]
```diff
@@ -135,9 +155,9 @@ def get_workers(settings):
             parallel_tests=40,
         ),
         cpw(
-            name="cstratak-RHEL8-fips-x86_64",
```
Comment (vstinner) [inline master/custom/workers.py:138]: Did you remove this config on purpose?

Comment (stratakis) [reply master/custom/workers.py:138]: Moved it up together with the other x86_64 ones.

**Comment by vstinner** [comment]
Merged, thank you ;-)

**Comment by vstinner** [comment]
test_json and test_lib2to3 started to crash on s390x Fedora Clang 3.x buildbot: https://bugs.python.org/issue46542

Can it be related to this change?

**Pull request merged**